### DISCUSSION TITLE
I want to understand how much manual intervention there has been in the samples, but I cannot find such info.

### DISCUSSION THREAD

**Discussion by mavavilj** [bytedeco/javacpp-presets#1325] [General]
I want to understand how much manual intervention there has been in the samples, but I cannot find such info.

If I was trying to sell JavaCPP, then I would expect that the users wish to know, how much effort might be required or whether their library is in principle even compatible. The front page suggested that it might fail on some "convoluted" libraries.

**Comment by saudet** [comment]
Please refer to https://github.com/bytedeco/javacpp-presets/wiki/Create-New-Presets
